ocrd/ocrd/resource_manager.py

```diff
@@ -84,6 +84,8 @@ def list_available(self, executable=None):
         List models available for download by processor
         """
         if executable:
+            if executable not in self.database:
+                return []
             return [(executable, self.database[executable])]
```

Comment:
```suggestion
            return [(executable, self.database.get(executable, []])]
```

We need to return the same format, a list of tuples `[(executable, list_of_resources)]`, returning just an empty list is not enough.

Reply:
@kba I don't think that's true. If the returned list is empty, then nothing will get printed (which is probably what you want when using `ocrd resmgr list-available -e ...` for other tools). In your variant `ocrd.cli.print_resources` will print the name of the processor, followed by an empty line.

Reply:
Yes, that's why I meant that we should adapt `print_resources` to print something like

```
ocrd-foo-bar: NONE
```

or similar.